spec/hcloud/ssh_key_spec.rb

```diff
@@ -108,7 +108,7 @@
     expect(client.ssh_keys.first).to be_a Hcloud::SSHKey
     id = client.ssh_keys.first.id
     expect(id).to be_a Integer
-    expect(client.ssh_keys.find(id).destroy).to be true
+    expect(client.ssh_keys.find(id).destroy).to be_a Hcloud::SSHKey
```

Comment:
As discussed it's a bit confusing that I get an (Key) object as result for a destroy operation which is in my eyes a "boolean". It either works or it doesn't.